test/Libraries/RevitTestServices/RevitSystemTestBase.cs

```diff
@@ -363,12 +363,44 @@ protected static void SwapCurrentModel(string modelPath)
             initialDoc.Close(false);
         }
 
+        #endregion
+
+        protected void MakeConnector(NodeModel start, NodeModel end, int portStart, int portEnd)
+        {
+            var cmdStart = new DynamoModel.MakeConnectionCommand(start.GUID, portStart, PortType.Output,
+                DynamoModel.MakeConnectionCommand.Mode.Begin);
+            this.Model.ExecuteCommand(cmdStart);
+
+            var cmdend = new DynamoModel.MakeConnectionCommand(end.GUID, portEnd, PortType.Input,
+                DynamoModel.MakeConnectionCommand.Mode.End);
+            this.Model.ExecuteCommand(cmdend);
+        }
+      
+        private static void UpdateSystemPathForProcess()
+        {
+            var assemblyLocation = Assembly.GetExecutingAssembly().Location;
+            var assemblyDirectory = Path.GetDirectoryName(assemblyLocation);
+            var parentDirectory = Directory.GetParent(assemblyDirectory);
+            var corePath = parentDirectory.FullName;
+
+            // Add the main exec path to the system PATH
+            // This is required to pickup certain dlls.
+            var path =
+                Environment.GetEnvironmentVariable(
+                    "Path",
+                    EnvironmentVariableTarget.Process) + ";" + corePath;
+            Environment.SetEnvironmentVariable("Path", path, EnvironmentVariableTarget.Process);
+        }
+    }   
+
+    public class RevitSystemTestHelper
```

Comment:
Could you move this class to a seprate file?

Reply:
I have a new commit, could you help review it?